src/js/components/sort_option.js

```diff
@@ -5,15 +5,14 @@ Vue.component('sort-option', {
     label: String,
   },
   template: `
-    <label v-bind:for="id">
-      <input type="radio"
-             name="sort"
-             v-bind:id="id"
-             v-bind:value="order"
-             v-model="sort"
-             v-on:change="$emit('sort-candidates', order)">
-      {{ label }}
-    </label>
+  <div class="search_sortOption">
+    <input class="search_sortOptionInput" type="radio" name="sort"
+              v-bind:id="id"
```

Comment:
If multi-line, would you mind if we have one attribute per line? (`type` and `name` can be moved down)

Reply:
It looks good even show the full name~ so I'll change it back~